flow/designs/asap7/mock-array/config.py

```diff
@@ -1,47 +1,34 @@
 import os
 
-# number of Elements in row and column, user can set via environment variable
-rows = int(os.environ.get("MOCK_ARRAY_HEIGHT"))
-cols = int(os.environ.get("MOCK_ARRAY_WIDTH"))
-
-# Element placement pitch can be control by user
-pitch_scale = int(os.environ.get("MOCK_ARRAY_PITCH_SCALE"))
-
-if pitch_scale < 1:
-  raise Exception("Element placement pitch must be greater than 1")
-
-# Routing pitches for relevant metal layers.
-#  For x, this is M5; for y, this is M4.
```

Comment:
this comment with the explanation of the 0.048 magic value is valuable, I'd like to keep it. 2.16 is a multiple of 0.048...

Reply:
Will do.

Please note - I am chasing the HEAD so I can get other work done.
Will implement / comment once I have this baseline pushed thru...